### ISSUE TITLE
Setup fails due to website dependency

### ISSUE THREAD

**Issue by dhmay** [cproctor/scratch_hue#2]
When I run **scratch_hue_helper**, I get the trace below. It looks like there's a dependency on http://mrproctor.net/scratch_extension_policy, which is currently timing out. I'm excited to try this out!

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/bin/scratch_hue_helper", line 51, in <module>
    if get_policy('hue') != 'enabled':
  File "/Library/Frameworks/Python.framework/Versions/2.7/bin/scratch_hue_helper", line 47, in get_policy
    response = requests.get('http://mrproctor.net/scratch_extension_policy')
  File "/Library/Python/2.7/site-packages/requests/api.py", line 69, in get
    return request('get', url, params=params, **kwargs)
  File "/Library/Python/2.7/site-packages/requests/api.py", line 50, in request
    response = session.request(method=method, url=url, **kwargs)
  File "/Library/Python/2.7/site-packages/requests/sessions.py", line 468, in request
    resp = self.send(prep, **send_kwargs)
  File "/Library/Python/2.7/site-packages/requests/sessions.py", line 576, in send
    r = adapter.send(request, **kwargs)
  File "/Library/Python/2.7/site-packages/requests/adapters.py", line 423, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='mrproctor.net', port=80): Max retries exceeded with url: /scratch_extension_policy (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x1031af990>: Failed to establish a new connection: [Errno 60] Operation timed out',))

**Comment by cproctor** [comment]
I just pushed a change that fixes this issue. Have fun! I'd love to know what you do with this :)

This issue has a little bit of back story. When I wrote this, I was teaching middle-school CS, and we had installed Hue lights across the whole classroom ceiling. I wanted my students to experience the feeling that the world is scriptable by letting them do whatever they wanted with the classroom lights. I also wanted to be able to disable their experiments if they got annoying. Finally, I wanted this security system to be sufficiently straightforward that a motivated student could understand and circumvent it. http://mrproctor.net/scratch_extension_policy was a small JSON file that indicated that the script should be enabled or disabled. `scratch_hue_helper` fetches the file and checks whether to exit. Unfortunately, I'm not teaching right now and haven't kept up mrproctor.net.
